web/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/web/rest/person/PersonController.kt

```diff
@@ -133,6 +135,52 @@ class PersonController @Autowired constructor(
     ) {
         val sammensatt = listOfNotNull(fornavn, mellomnavn, etternavn).joinToString(" ")
     }
+    data class DeltBostedDTO(
+        val startdatoForKontrakt: String?,
+        val sluttdatoForKontrakt: String?,
+        val adresse: AdresseDTO?,
+        val ukjentBosted: UkjentBostedDTO?
+    )
+
+    data class UkjentBostedDTO(
+        val bostedskommune: String?
+    )
+
+    data class AdresseDTO(
+        val adressenavn: String?,
+        val husnummer: String?,
+        val husbokstav: String?,
+        val bruksenhetsnummer: String?,
+        val kommunenummer: String?,
+        val postnummer: String?,
+        val bydelsnummer: String?,
+        val tilleggsnavn: String?,
+        val coAdressenavn: String?
+    )
+
+    private fun hentDeltBosted(deltBosted: List<HentPerson.DeltBosted>): List<DeltBostedDTO> {
+        return deltBosted.map {
+            DeltBostedDTO(
+                startdatoForKontrakt = it.startdatoForKontrakt?.value?.format(ISO_DATE_TIME),
+                sluttdatoForKontrakt = it.sluttdatoForKontrakt?.value?.format(ISO_DATE_TIME),
+                adresse = AdresseDTO(
```

Comment:
Fant vi ut noe om hvorvidt `vegadresse` og `matrikkeladresse` kunne eksistere samtidig?
Uten å ha kjennskap til det, så er jeg litt skeptisk til at vi fyller ut f.eks `postnummer` på denne måten

Reply:
Nei, kan ta en prat med PDL for å høre om de har noe kunnskap til det. Enig i at her gjetter vi litt for mye på noe som er litt sårbart ut mot bruker

Reply:
Så den andre kommentaren din om at `vegadresse` og `matrikkeladresse` ikke ville være definert samtidig. 
Så da er ikke dette ett problem. :+1: